pytorch_lightning/trainer/trainer.py

```diff
@@ -261,6 +262,9 @@ def __init__(
 
             track_grad_norm: -1 no tracking. Otherwise tracks that p-norm. May be set to 'inf' infinity-norm.
 
+            track_grad_norm_mode: How grad norms are aggregated. By default it aggregates for each parameter.
+                Additionally it can aggregate for parameter+optimizer and optimizer.
```

Comment:
```suggestion
                The possible options are `parameters`, `optimizer`, `optimizer+parameters`.
```

Reply:
Do you have a suggestion for better naming here? `optimizer+parameters` in particular is not a good name from a user perspective.